src/main/resources/hudson/security/table.js

```diff
@@ -222,7 +222,7 @@ function getTooltipAttributeName() {
  */
 Behaviour.specify(".global-matrix-authorization-strategy-table td input", 'GlobalMatrixAuthorizationStrategy', 0, function(e) {
   var table = findAncestor(e, "TABLE");
-  if (table.hasClassName('read-only')) {
+  if (table.classList.contains('read-only')) {
```

Comment:
Identify SystemRead / ExtendedRead config forms and keep all checkboxes disabled.